code/LanguagePage.ts

```diff
@@ -112,7 +112,7 @@ pipeTable
  )}
 `
   }
-
+  
```

Comment:
can you undo changes to this file? They are inconsequential but just to keep it a cleaner PR

Reply:
I tried it, but it still detects some change... almost smoke I would say. I don't know if it's possible that I don't see any change.